functions/__bobthefish_glyphs.fish

```diff
@@ -62,6 +62,7 @@ function __bobthefish_glyphs -S -d 'Define glyphs used by bobthefish'
     set virtualenv_glyph \uE73C ' '
     set ruby_glyph       \uE791 ' '
     set go_glyph         \uE626 ' '
+    set node_glyph       \uF898 ' '
```

Comment:
let's use `\uE718` instead? they look like they're the same, but `\uE718` has been in the fonts for a lot longer, so there's a better chance that existing users will have it in their patched fonts.

Reply:
Sure. Was not aware of the fact; thank you.